html/forms/html-form-structure/payment-form.html

```diff
@@ -8,87 +8,96 @@
-    <p>Required fields are followed by <strong><span aria-label="required">*</span></strong>.</p>
-    <section>
-        <h2>Contact information</h2>
-        <fieldset>
-          <legend>Title</legend>
-          <ul>
-              <li>
-                <label for="title_1">
-                  <input type="radio" id="title_1" name="title" value="A">
-                  Ace
-                </label>
-              </li>
-              <li>
-                <label for="title_2">
-                  <input type="radio" id="title_2" name="title" value="K" >
-                  King
-                </label>
-              </li>
-              <li>
-                <label for="title_3">
-                  <input type="radio" id="title_3" name="title" value="Q">
-                  Queen
-                </label>
-              </li>
-          </ul>
-        </fieldset>
-        <p>
-          <label for="name">
-            <span>Name: </span>
-            <strong><span aria-label="required">*</span></strong>
-          </label>
-          <input type="text" id="name" name="username">
-        </p>
-        <p>
-          <label for="mail">
-            <span>E-mail: </span>
-            <strong><span aria-label="required">*</span></strong>
-          </label>
-          <input type="email" id="mail" name="usermail">
-        </p>
-        <p>
-          <label for="pwd">
-            <span>Password: </span>
-            <strong><span aria-label="required">*</span></strong>
-          </label>
-          <input type="password" id="pwd" name="password">
-        </p>
-    </section>
-    <section>
-        <h2>Payment information</h2>
-        <p>
-          <label for="card">
-            <span>Card type:</span>
-          </label>
-          <select id="card" name="usercard">
-            <option value="visa">Visa</option>
-            <option value="mc">Mastercard</option>
-            <option value="amex">American Express</option>
-          </select>
-        </p>
-        <p>
-          <label for="number">
-            <span>Card number:</span>
-            <strong><span aria-label="required">*</span></strong>
-          </label>
-            <input type="tel" id="number" name="cardnumber">
-        </p>
-        <p>
-          <label for="expiration">
-            <span>Expiration date:</span>
-            <strong><span aria-label="required">*</span></strong>
-          </label>
-          <input type="text" id="expiration" required="true" placeholder="MM/YY" pattern="^(0[1-9]|1[0-2])\/([0-9]{2})$">
-        </p>
-    </section>
-    <section>
-        <p> <button type="submit">Validate the payment</button> </p>
-    </section>
+<p>
+  Required fields are followed by
+  <strong><span aria-label="required">*</span></strong>.
+</p>
+<section>
+  <h2>Contact information</h2>
+  <fieldset>
+    <legend>Title</legend>
+    <ul>
+      <li>
+        <label for="title_1">
+          <input type="radio" id="title_1" name="title" value="K" />
```

Comment:
Input names should not be[ identical](https://github.com/sharon-kathambi/learning-area/pull/2/files#diff-28166d2eeea26155315fe19e41bb79f469f35574afa3c483b66db41ffc381e3eR30)